Tasks/AzureWebPowerShellDeployment/task.loc.json

```diff
@@ -13,7 +13,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 34
+    "Patch": 35
```

Comment:
I have bumped up tasks which are using VstsAzureHelpers_ module, Do I need to bump up all tasks using common module.